Pokepedia.Domain/Entities/Pokemons/Pokemon.cs

```diff
@@ -9,13 +9,19 @@ public class Pokemon
         public string Name { get; }
         public int Weight { get; }
         public int Order { get; }
+        public string LocationArea { get; }
+        public string LocationDetails { get; }
+
```

Comment:
Perhaps this should be its own thing? LocationData? instead of Pokemon? Also remove your extra spaces